users/itousae/4.js

```diff
@@ -0,0 +1,7 @@
+function output(array) {
+  const result = array.filter((el) => el % 2 !== 0);
+  console.log(result);
```

Comment:
```suggestion
  return array.filter((el) => el % 2 !== 0);
```

Reply:
Thank you for your confirmation.
I made a mistake and committed a few times...

Fixed. ce742afecaed84c77d201cc4c74c46e8b2078f31